includes/Experiments.php

```diff
@@ -383,6 +383,17 @@ public function get_experiments(): array {
 				'description' => __( 'Enable recording from webcam/microphone', 'web-stories' ),
 				'group'       => 'editor',
 			],
+			/**
+			 * Author: @spacedmonkey
+			 * Issue: #11081
+			 * Creation date: 2022-03-30
+			 */
+			[
+				'name'        => 'captionHotlinking',
+				'label'       => __( 'Caption hotlinking', 'web-stories' ),
+				'description' => __( 'Enable hotlinking of captions', 'web-stories' ),
+				'group'       => 'general',
```

Comment:
```suggestion
				'group'       => 'editor',
```

Since this only applies to the editor, not to both editor and dashboard.